chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -27,31 +27,31 @@ questionableActivityNoDataEmailBody=<p>No questionable activity was found betwee
 #pending change request email properties
 pendingChangeRequestReportFilename=pending-change-request
 pendingChangeRequestEmailSubject=Pending Change Request Report
-pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p><p><ul><li>%s pending Change Website Requests were found</li></ul></p>
-pendingChangeRequestNoDataEmailBody=<p>No pending change requests were found.</p>
+pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p><p><ul><li>%s pending Change Website Requests were found</li></ul></p>Report contains data for the following ONC-ACBs:
+pendingChangeRequestNoDataEmailBody=<p>No pending change requests were found.</p>Report contains data for the following ONC-ACBs:
 
 #oversight daily email properties
 oversightEmailDailySubject=ONC Overnight Broken Surveillance Rules Report
-oversightEmailDailyHtmlMessage=<h3>ONC Overnight Broken Surveillance Rules Report</h3>
+oversightEmailDailyHtmlMessage=<h3>ONC Overnight Broken Surveillance Rules Report</h3>Report contains data for the following ONC-ACBs:
```

Comment:
In the pending change request emails, the list of ACBs appears as sort of the last thing in the message, but here I think it will appear first right under a heading.  Can we pick one place and stick with it? I would vote for putting the ACB list first, even if some other text needs re-wording, as is done here.